fossunited/chapters/doctype/foss_chapter_event/foss_chapter_event.py

```diff
@@ -111,6 +111,25 @@ def before_save(self):
             self.update_published_status()
         self.set_route()
 
+    def validate(self):
+        self.validate_permalink()
+
+    def validate_permalink(self):
+        if self.is_external_event:
+            return
+
+        if frappe.db.exists(
+            self.doctype, {"event_permalink": self.event_permalink}
+        ):
+            frappe.throw(
```

Comment:
This sort of takes into account for every save - when I save a permalink, and then makes another change, and then save - the permalink check raises an `Exists Error` 
![permalink-error](https://github.com/user-attachments/assets/4b38b424-bd46-4250-a6ea-c18338a7c4b5)